cypress.config.ts

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
   chromeWebSecurity: false,
   experimentalMemoryManagement: true, // better memory management, see https://github.com/cypress-io/cypress/pull/25462
   retries: { runMode: 2 },
+  videoCompression: false,
```

Comment:
what are the tradeoffs here? lower cpu utilization vs bandwidth cost? (are we charged for bandwidth?)

Reply:
afaict cypress does not charge for storage, so the tradeoff is in github actions. I have no transparency into github actions, but I'd strongly prefer better videos of failures so it's easier to debug them.

Reply:
this only impacts the final step, uploading tests, so it only affects runtime if there are flakes or failures.